snap/snapcraft.yaml

```
@@ -15,6 +15,13 @@ architectures:
 parts:
   heroku:
     plugin: nil
+    override-pull: |
+      snapcraftctl pull
+      COMMIT_MSG=$(git log --pretty=oneline --abbrev-commit | head -n 1 | awk '{print $2}')
+      CLI_VERSION=$(grep \"version\": packages/cli/package.json | head -n 1 | awk '{print $2}' | tr -d \" | tr -d \,)
```

I am not sure what we have available to run at this step, but if we have [node available](https://github.com/heroku/cli/blob/2715f9ccd686e654f075d8b7af38d3fc69803fdb/snap/snapcraft.yaml#L28), we could use node to generate this instead: 

```suggestion
      CLI_VERSION=$(node -e 'console.log(require("./packages/cli/package.json").version)')
```

---

I don't believe node is available at this step in the build (pull step)